environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
 - sympy=1.0
 - tensorflow=1.1.0
 - nodejs=6.11.0=0
+- yaml
```

Comment:
Should be `pyyaml`?  It looks like `pyyaml` was picked up by one of the other requirements, but it couldn't hurt to make it explicit here.